reana_server/rest/workflows.py

```diff
@@ -997,6 +1002,7 @@ def start_workflow(workflow_id_or_name, user):  # noqa
                 .get("workflow", {})
                 .get("type", None)
             )
+            workflow.inactivate_workspace_retention_rules()
```

Comment:
What is your opinion on moving this line inside `clone_workflow`? In this way, all the logic to clone a workflow is in a single function